pyant/instances/__init__.py

```diff
@@ -88,6 +88,8 @@ class BeamInstancesGetter:
     __all__ = beam_instances
 
     def __getattr__(self, name):
+
+        print(f'getter looking for {name}')
```

Comment:
There should generally not be any printing outputs produced by libraries, there could however be calls to a logger

Reply:
Understood and agreed. This line was never meant to be permanent.

Reply:
:+1: